.hlint.yaml

```diff
@@ -28,6 +28,11 @@
   - {name: [Data.Set, Data.HashSet], as: Set}
   - {name: [Data.Map, Data.HashMap.Strict, Data.HashMap.Lazy], as: Map}
   - {name: Control.Arrow, within: []}
+  - {name: HypotheticalModule1, as: HM1, asRequired: true}
```

Comment:
This file should not be used for testing. You can create a separate yaml file under `data`, and use it in a `test` file under `tests`. See `data/wildcard.yaml` and `tests/wildcard.test` for an example.